Plugins/GitUIPluginInterfaces/ObjectId.cs

```diff
@@ -21,17 +21,17 @@ public sealed class ObjectId : IEquatable<ObjectId>, IComparable<ObjectId>
         /// <summary>
         /// Gets the artificial ObjectId used to represent working directory tree (unstaged) changes.
         /// </summary>
-        public static ObjectId WorkTreeId { get; } = new(0x11111111, 0x11111111, 0x11111111, 0x11111111, 0x11111111);
+        public static ObjectId WorkTreeId { get; } = new(0x1111111111111111, 0x1111111111111111, 0x11111111);
```

Comment:
```suggestion
        public static ObjectId WorkTreeId { get; } = new(0x11111111_11111111, 0x11111111_11111111, 0x11111111);
```

Reply:
OK in groups with 8 or 4 digits?
(did not know about this feature)

Reply:
> in groups with 8 or 4 digits?

As you prefer.

Reply:
group by 4